apps/polyflix/src/modules/core/components/Jumbotron/jumbotron.component.tsx

```diff
@@ -0,0 +1,276 @@
+  [theme.breakpoints.down('md')]: {
+    marginBottom: `${theme.spacing(2)} !important`,
+  },
+  [theme.breakpoints.up('md')]: {
+    transform: 'translateY(50%)',
+  },
+}))
+
+type SearchBarProps = {
+  sx?: SxProps<Theme>
+}
+
+export const HomeSearchBar = ({ sx }: SearchBarProps) => {
+  return (
+    <HomeSearchBarRootStyled variant="outlined" sx={sx}>
+      <InputBase
+        sx={{ ml: 1, flex: 1, width: '100%' }}
+        placeholder="Search"
+        inputProps={{ 'aria-label': 'search' }}
+      />
+      <IconButton type="button" sx={{ p: '10px' }} aria-label="search">
+        <SearchIcon />
+      </IconButton>
+    </HomeSearchBarRootStyled>
+  )
+}
+
+const StyledSwiper = styled(Swiper)(() => ({
+  '& .swiper-pagination-bullet.swiper-pagination-bullet-active': {
+    background: 'red !important',
+  },
+}))
+
+const JumbotronMainTile = () => {
+  const { t } = useTranslation('common')
+
+  const { data, isLoading } = useGetVideosQuery({
+    visibility: Visibility.PUBLIC,
+    draft: false,
+    order: '-views',
+    page: 1,
+    pageSize: 5,
+  })
+
+  if (isLoading) {
+    return (
+      <Box
+        sx={{
+          borderRadius: 1,
+          overflow: 'hidden',
+        }}
+      >
+        <AspectRatioBox ratio={16 / 9}>
+          <Skeleton variant="rectangular" width="100%" height="100%" />
+        </AspectRatioBox>
+      </Box>
+    )
+  }
+
+  if (!isLoading && !data?.items?.length) {
+    return (
+      <AspectRatioBox ratio={16 / 9}>
+        <Alert severity="info">{t('noData.title')}</Alert>
+      </AspectRatioBox>
+    )
+  }
+
+  return (
+    <Box
+      sx={{
+        borderRadius: 1,
+        overflow: 'hidden',
+      }}
+    >
+      <AspectRatioBox ratio={16 / 9}>
+        <StyledSwiper
+          spaceBetween={30}
+          centeredSlides={true}
+          loop={true}
+          autoplay={{
+            delay: 5000,
+            disableOnInteraction: false,
+          }}
+          pagination={{
+            clickable: true,
+          }}
+          modules={[EffectFade, Autoplay, Pagination]}
+        >
+          {(data?.items || []).map((element, i) => (
+            <SwiperSlide key={i}>
+              <Box
+                sx={{
+                  position: 'relative',
+                }}
+              >
+                <ImageCover ratio={16 / 9} src={element.thumbnail || ''} />
+                <Typography
+                  variant="body1"
+                  color="Background"
+                  sx={{
+                    position: 'absolute',
+                    left: 8,
+                    bottom: 8,
+                    background: (theme) =>
+                      alpha(theme.palette.common.black, 0.7),
+                    px: 2,
+                    py: 0.5,
+                    borderRadius: 1,
+                  }}
+                >
+                  {element.title}
+                </Typography>
+              </Box>
+            </SwiperSlide>
```

Comment:
It's seems that we cannot click on the carousel slide to jump to the video, right ?

Reply:
True, i fix it :up: